src/server/controlchan.rs

```diff
@@ -69,14 +69,10 @@ impl Encoder for FTPCodec {
             }
             Reply::MultiLine { code, mut lines } => {
                 let s = lines.pop().unwrap();
-                write!(
```

Comment:
Maarten I realized after merging this breaks other mult-line responses since it adds the "END" line to it.

From: RFC 959 :

```
There will be cases however, where the text is longer than
      a single line.  In these cases the complete text must be bracketed
      so the User-process knows when it may stop reading the reply (i.e.
      stop processing input on the control connection) and go do other
      things.  This requires a special format on the first line to
      indicate that more than one line is coming, and another on the
      last line to designate it as the last.  At least one of these must
      contain the appropriate reply code to indicate the state of the
      transaction.  To satisfy all factions, it was decided that both
      the first and last line codes should be the same.

         Thus the format for multi-line replies is that the first line
         will begin with the exact required reply code, followed
         immediately by a Hyphen, "-" (also known as Minus), followed by
         text.  The last line will begin with the same code, followed
         immediately by Space <SP>, optionally some text, and the Telnet
         end-of-line code.

            For example:
                                123-First line
                                Second line
                                  234 A line beginning with numbers
                                123 The last line

         The user-process then simply needs to search for the second
         occurrence of the same reply code, followed by <SP> (Space), at
         the beginning of a line, and ignore all intermediary lines.  If
         an intermediary line begins with a 3-digit number, the Server
         must pad the front  to avoid confusion.
```

Reply:
I'll make a PR for this.

Reply:
See https://github.com/bolcom/libunftp/pull/50/files, I've merged it, feel free to comment after the fact. I still want to add some unit test.